docker.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# plugin to set "dnf" proxy settings for ProxyMan
+# privileges has to be set by the process which starts this script
+
+CONF_FILE="/etc/systemd/system/docker.service.d/http-proxy.conf"
+
+
+list_proxy() {
+    # inefficient way as the file is read twice.. think of some better way
+    echo -e "${bold}docker proxy settings: ${normal}"
+    if [ ! -e "$CONF_FILE" ]; then
+        echo -e "${red}None${normal}"
+        return
+    else
+        lines="$(cat $CONF_FILE | grep proxy -i | wc -l)"
+        if [ "$lines" -gt 0 ]; then
+            cat $CONF_FILE | grep proxy -i | sed -e "s/Environment=//g" -e "s/\_/\ /g"
+        else
+            echo -e "${red}None${normal}"
+        fi
+    fi
+}
+
+unset_proxy() {
+    if [ ! -e "$CONF_FILE" ]; then
+        return
+    fi
+    if [ "$(cat $CONF_FILE | grep proxy -i | wc -l)" -gt 0 ]; then
+        sudo rm $CONF_FILE
+        sudo service docker restart
+    fi
+}
+
+set_proxy() {
+    unset_proxy
+    if [ ! -e "$CONF_FILE" ]; then
```

Comment:
Need to ensure the directory exists.
Something like `mkdir -p /etc/systemd/system/docker.service.d` should work.